lab-cara/main.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const reader = require('./lib/reader');
+
+const georgiaPath = `${__dirname}/data/georgia.txt`;
+const missouriPath = `${__dirname}/data/missouri.txt`;
+const washingtonPath = `${__dirname}/data/washington.txt`;
+
+const dataFiles = [georgiaPath, missouriPath, washingtonPath];
+const results = [];
+
+const readDataFilesArrayAsync = (dataFiles, currentIndex, callback) => {
+  if (currentIndex >= dataFiles.length) {
+    return callback();
+  }
+  const currentFilePath = dataFiles[currentIndex];
+  try {
+    return reader.readDataFilesAsync(currentFilePath, (file) => {
```

Comment:
You forgot to handle the error here, and null from reader.js is being passed in for file.